CHANGES.md

```diff
@@ -18,6 +18,7 @@
   a custom distrib archive file. (#255, @NathanReb)
 - Use int64 for timestamps. (#261, @gpetiot)
 - Define the order of packages (#263, @gpetiot)
+- Define default values for dry-run mode (#262, @gpetiot)
```

Comment:
That's not very helpful to users, I think it should instead mention how it improves the dry-run mode by allowing it to continue even after some API call's response were expected by using placeholder values.

Reply:
done